src/main/java/org/snomed/snowstorm/core/data/domain/Annotation.java

```diff
@@ -0,0 +1,137 @@
+package org.snomed.snowstorm.core.data.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonView;
+import io.micrometer.core.instrument.util.StringUtils;
+import org.snomed.snowstorm.core.pojo.TermLangPojo;
+import org.snomed.snowstorm.rest.View;
+
+import java.util.Objects;
+import java.util.UUID;
+
+@JsonView(value = View.Component.class)
+@JsonIgnoreProperties({"memberId", "additionalFields", "effectiveTimeI", "deleted", "changed"})
+public class Annotation extends ReferenceSetMember implements AnnotationView {
+
+	private String annotationId;
+	private String annotationTypeId;
+	private ConceptMini annotationType;
+	private String annotationValue;
+	private String annotationLanguage;
```

Comment:
@QuyenLy87, going by the ticket, it looks like `annotationTarget` is missing. Do you know if this field is needed or is it an old requirement?

Reply:
We're discussing tomorrow whether or not annotations that point to concepts should be a thing.  Personally I think that non-defining relationships are the better solution there.

Reply:
Hi @dmcgihtsdo, I'm not sure about the annotationTarget. But at the moment, we don't store it in ES and also it's not used by UI